doc/misc/examples/obsolete/rebalance.py

```diff
@@ -50,15 +50,31 @@ class RebalanceProgram:
     """)
     parser = optparse.OptionParser(usage)
     parser.add_option(
-        "-d", "--dirs", dest="dirs", type="int", default=22, help=dedent("""\
-        Number of directories among which to balance (default=%default)."""))
+        "-d",
```

Comment:
I have inspected files in this subtree:
```
doc/misc
```
And I think it's nearly 100% obsolete. Besides, the subtree is not used for generating documentation on Qserv. See:
```
doc/index.rst
```
I vote for eliminating the subtree altogether.